### PR TITLE
Update appdata/desktop from eos-shell-content

### PR DESCRIPTION
**Pull request by ramcq**
This means the Flatpak will build normally using flatpak-builder
without needing any post-processing by special Endless tools.

https://phabricator.endlessm.com/T23463

### CODE REVIEW

File: org.mozilla.Firefox.appdata.xml [hunk 1]
```diff
@@ -1,22 +1,65 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <id>org.mozilla.Firefox.desktop</id>
+  <metadata_license>CC0</metadata_license>
```
Comment (nedrichards) [inline org.mozilla.Firefox.appdata.xml]: CC0-1.0

**Comment by nedrichards** [comment]
Could you add OARS tags to the appdata (this means we won't have to change anything for parental controls). I believe the following policy is descriptive:
```
  <content_rating type="oars-1.1">
    <content_attribute id="social-chat">intense</content_attribute>
    <content_attribute id="social-info">intense</content_attribute>
    <content_attribute id="social-audio">intense</content_attribute>
  </content_rating>
```

**Comment by ramcq** [comment]
OARS updated.

**Review by nedrichards — APPROVED** [review]
:1st_place_medal:

**Pull request merged**